### ISSUE TITLE
Could ICECC_PREFERRED_HOST be updated to support multiple hosts

### ISSUE THREAD

**Issue by alanbithell** [icecc/icecream#615]
I sometimes find that a device on my network may start to misbehave and break my build with `broken pipe` errors, so I end up having to resort to using `ICECC_PREFERRED_HOST` to select a know good device to help speed up my compile jobs at least a little while I try to track down the user of that device.

Unfortunately this is happening more and more these days with multiple machines sitting behind a router with both machines running `iceccd` but port forwarding only reaching one of them.

If I had the ability to select multiple preferred hosts, I'd probably just lock that into my config and be done with it. Maybe it would be better to have a config for `ICECC_IGNORED_HOSTS` or similar.

**Comment by coolo** [comment]
ignored_hosts sounds like the more convincing user story.

**Comment by alanbithell** [comment]
Agreed `ICECC_IGNORED_HOST` sounds like a better option. Initially I was thinking of selecting a golden sample of machines I know and trust. But ignoring the bad actors would be better.
